synapse/lib/layer.py

```diff
@@ -158,14 +158,14 @@ async def storNodeEditsNoLift(self, nodeedits, meta=None):
 
         await self.layr.storNodeEditsNoLift(nodeedits, meta)
 
-    async def syncNodeEdits(self, offs, wait=True):
+    async def syncNodeEdits(self, offs, wait=True, reverse=False):
```

Comment:
why is this not added to syncNodeEdits2 API?

Reply:
It is. But this API is used by the `$layer.edits()` API so we need to plumb both.